### PR TITLE
added search functionalities and loaders

### PR DESCRIPTION
**Pull request by Rohanmond**
Added Search functionalities and loader

preview=> https://fashi-fy-react-7yls80h9q-rohanmond.vercel.app/

### CODE REVIEW

File: fashify/src/utils/utils.js [hunk 1]
```diff
@@ -54,11 +54,17 @@ const sortByPrice = (data, sortCat) => {
 const priceRangeFilter = (data, maxValue) => {
   return data.filter((el) => Number(el.price) <= maxValue);
 };
-
+const searchFilter = (data, keyword) => {
+  if (keyword === "") return data;
+  return data.filter((el) =>
+    el.title.toLowerCase().startsWith(keyword.toLowerCase())
+  );
+};
```
Comment (romabulani) [inline fashify/src/utils/utils.js:62]: Instead of startsWith you can use includes function, then it will search in whole title

**Review by romabulani — APPROVED** [review]
Very clean and self explanatory code
I Learnt the use constants.js file, will refactor my code to use this.

**Review by shrey27 — APPROVED** [review]
- Overall code looks good
- Preview is also working fine
- looks good to merge

**Review by anuj-kumary — APPROVED** [review]
Clean code
Good naming conventions
Good file structure

File: fashify/src/utils/utils.js (hunk 1 above)
Comment (Rohanmond) [reply fashify/src/utils/utils.js:62]: I actually used that but putting men it was not discarding women . So I used startWith here. Thanks for your comment

**Pull request merged**